learn/getting-started.md

```diff
@@ -4,23 +4,18 @@ title: Getting Started
 permalink: /learn/getting-started/
 ---
 
-# Getting Started
+# Installing Ballerina
```

Comment:
Here we are changing the topic from "Getting Started" -> "Installing Ballerina". So shouldn't we change the file name and url path too to be consistent with other pages. In that case we may need to redirect old url to this one as well.

Reply:
Renamed the "getting-started.md" file to "installing-ballerina.md".